mautic/mautic
# M3: Page bundle upgrade

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/8005
| BC breaks? | Y
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Remove deprecated code from PageBundle

Slideshows for pages marked as deprevcated for now. Nothing found in official DOC.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com/)
2. Ensure that configuration for landing pages works as before
3. Ensure that form for pages is working as before
4. Test form for A/B test pages 
5. Ensure that you can set point action for landing page hit and it works.
5. Ensure that you can set point action for url hit and it works.
6. Ensure that you can add page hit action in campaign builder and it work like before.
7. Check page visits dashboard widget config form
8. Check that Preference center page field works in Email form
9. Test that page builder works as before

#### List backwards compatibility breaks:
1. AbTestHelper class removed, use DetermineWinnerSubscriber instead
2. Removed Mautic\PageBundle\Form\Type\SlideshowGlobalConfigType
3. Removed Mautic\PageBundle\Form\Type\SlideshowSlideConfigType
